elections/committee-conflict-of-interest.md

```diff
@@ -0,0 +1,47 @@
+# Committee Conflicts of Interest
+
+## Steering Committee and Code of Conduct Committee
+
+At time of writing, the only known conflict of interest for serving on
+committees in the Kubernetes Community exists with the
+[Steering Committee](/committee-steering/README.md) and
+the [Code of Conduct Committee (CoCC)](/committee-code-of-conduct/README.md).
+
+**_Kubernetes Community members cannot serve on both committees simultaneously._**
+
+This conflict may arise in the following scenarios:
+
+- A standing Steering Committee member has accepted a nomination to serve on
+  the Code of Conduct Committee
+- A standing CoCC member has elected to run for a seat on the Steering
+  Committee
+- A Kubernetes Community member has been nominated for candidacy for both
+  committees
+
+Given the uniqueness of the respective committee charters, their memberships'
+position in the community, and the difference in how these committees need to
+adjudicate on community matters, _candidates must be able to declare their
+preference in which committee they would like to serve on, if elected_.
+
+After the candidate is nominated for a position that would conflict and before
+the conclusion of the first of the two elections to complete, a non-conflicted
+Election Officer or in the event that a Steering Committee election is not
+active, a non-conflicted Steering Committee member will reach out to the
+candidate to determine which committee they want to continue with in the event
+that they are elected.
+
+This process should occur as soon as possible after a potential conflict has
+been identified.
+
+If the candidate is elected in the first election to conclude and that
+appointment matches their preference, the candidate will be withdrawn from the
+remaining open election.
+
+If applicable, the committee where that individual vacated their seat will fill
+that vacancy per existing election or committee vacancy rules.
+
+In the event that multiple candidates trigger this conflict in an election
+cycle, the two (2) affected candidates receiving the most votes will have their
+preference honored. In the event that the first election to conclude has only
+two seats under consideration, only the affected candidate with the most votes
+will have their preference honored.
```

Comment:
> In the event that the first election to conclude has only
two seats under consideration, only the affected candidate with the most votes
will have their preference honored.

The statement sounds like the choice of the candidate second most votes is affected by preference of the candidate with the highest votes. Why should there be a dependence?

Taking an example, let's assume Candidate A and Candidate B have won the elections to Committee 1, but both have prefer to be in Committee 2. Elections to Committee 2 happen in a month and they both intend to run for it.

If they both win the election to Committee 2 and they chose to vacate their seat in Committee 1, the rules for filling vacancy of Committee 1 would be triggered. In my opinion, I don't think in this case, we should restrict the candidate who got the second highest votes in election to Committee 2.

Reply:
Resolving the comment since the block doesn't exist in the latest version.